# HackBio_Cancer_internship
## Stage_Zero
### **RNA-Seq**

#### **Author: Mohammed Dahab (@M\_Dahab7)**

**Introduction**

RNA sequencing (RNA-Seq) leverages high-throughput sequencing to explore the transcriptome with greater coverage and resolution than previous methods. It enables the identification of novel transcripts, alternative splicing, and allele-specific expression across various RNA populations.(Kukurba & Montgomery, 2015)

**Body**

 RNA-seq, or RNA sequencing, utilizes high-throughput sequencing to identify the nucleotide sequence of RNA molecules and measure the abundance of different RNA species within a population. This analysis requires specialized computational tools to address sequencing errors, fragment biases, and other technological limitations. RNA-seq has led to significant scientific advancements, including discovering new therapies, a deeper understanding of genetic regulation, and identifying biomarkers and pathogenic mutations. The process of preparing an RNA-seq library begins with extracting and isolating RNA from a biological sample, such as a cell line or frozen tissue. For short-read sequencing, the isolated RNA is converted into cDNA through reverse transcription, then amplified using PCR and fragmented into short sequences. Once processed, the RNA-seq library is used as input for a sequencing platform to generate reads.(Deshpande et al., 2023)

 ![Screenshot 2024-09-03 000454](https://github.com/user-attachments/assets/b074b573-71d3-47ea-8ea5-b79d440a24cb)


<!--[if !mso]-->


 <!--[endif]-->Fig.1 overview main steps of rna-sequencing technology.(Deshpande et al., 2023)<!--[if !mso]--> 

<!--[endif]--> The main steps for RNA-seq data analysis include 1-Experimental Design, which involves selecting the appropriate library type, sequencing depth, and number of biological replicates. 2-Quality Control, assessing the quality of raw sequencing data to identify any issues that may affect downstream analysis. 3-Read Alignment, aligning the sequenced reads to a reference genome or transcriptome or performing de novo assembly if no reference is available. 4-Quantification of Gene and Transcript Levels. Measuring the expression levels of genes and transcripts from the aligned reads. 5-Differential Gene Expression Analysis. Identifying genes that show statistically significant differences in expression. 6-Functional Analysis. Characterizing the biological functions or pathways associated with differentially expressed genes (DEGs). 7-Visualization. Creating visual representations of the data, such as heat maps or volcano plots. 8-Advanced Analyses. This may include exploring alternative splicing, gene fusion detection, and integrating RNA-seq data with other functional genomics techniques.(Conesa et al., 2016)


**Conclusion**

In conclusion, RNA-seq is an essential tool for comprehensive transcriptome analysis, providing insights into gene expression under various conditions. The described bioinformatics workflow accurately identifies differentially expressed genes, enhancing our understanding of molecular mechanisms. By leveraging advanced computational tools, researchers can generate high-quality data, facilitating robust statistical analysis and meaningful biological interpretations.(Ji & Sadreyev, 2018)

**Reference**

Conesa, A., Madrigal, P., Tarazona, S., Gomez-Cabrero, D., Cervera, A., McPherson, A., Szcześniak, M. W., Gaffney, D. J., Elo, L. L., Zhang, X., & Mortazavi, A. (2016). A survey of best practices for RNA-seq data analysis. In _Genome Biology_ (Vol. 17, Issue 1). BioMed Central Ltd. https\://doi.org/10.1186/s13059-016-0881-8

Deshpande, D., Chhugani, K., Chang, Y., Karlsberg, A., Loeffler, C., Zhang, J., Muszyńska, A., Munteanu, V., Yang, H., Rotman, J., Tao, L., Balliu, B., Tseng, E., Eskin, E., Zhao, F., Mohammadi, P., P. Łabaj, P., & Mangul, S. (2023). RNA-seq data science: From raw data to effective interpretation. In _Frontiers in Genetics_ (Vol. 14). Frontiers Media S.A. https\://doi.org/10.3389/fgene.2023.997383

Ji, F., & Sadreyev, R. I. (2018). RNA-seq: Basic Bioinformatics Analysis. _Current Protocols in Molecular Biology_, _124_(1). https\://doi.org/10.1002/cpmb.68

Kukurba, K. R., & Montgomery, S. B. (2015). RNA sequencing and analysis. _Cold Spring Harbor Protocols_, _2015_(11), 951–969. https\://doi.org/10.1101/pdb.top084970

 
